ifrs17-template/Test/Data/IfrsVariableBenchmarks/BM_CH_2020_12_MTUP10pct_A.csv

```diff
@@ -3,12 +3,630 @@ Month,ReportingNode,Scenario,Year
 12,CH,MTUP10pct,2020
 @@A
 AccidentYear,AmountType,AocType,DataNode,EconomicBasis,EstimateType,Novelty,Value
-,NIC,CF,DT2.1,,A,C,-310
-2020,NIC,CF,DT1.2,,A,C,-310
+,,CF,DT1.1,,A,C,0
 ,NIC,CF,DT1.1,,A,C,-310
+,AEA,CF,DT1.1,,A,C,-5
+,ACA,CF,DT1.1,,A,C,-10
+,PR,CF,DT1.1,,A,C,400
+,,BOP,DT1.1,,A,I,0
+,NIC,BOP,DT1.1,,A,I,0
```

Comment:
yes, so that we can during testing we can check that the CE produces zeros for those figures. I think it is necessary.

Reply:
Another option would have been to cancel zeros from the benchmark and to exclude zeros from the tests as well. However, in this way I believe we would make the test weaker because we wouldn't check whether zeros are produced or not. Hence, I think the current setup for the benchmark/tests is the safest/strongest.

Reply:
It is just that we are using computation time to compare millions of 0. Yes, it is safe, I agree, but maybe there is a better solution to achieve the same, since this will be slow and will make BM files difficult to read.

Reply:
I hve createad an issue for this problem(https://github.com/Systemorph/IFRS17CalculationEngine/issues/132). We are aware that the current implementation is suboptimal, but the topic is quite delicate and need a dedicated PR,